### PR TITLE
With resizing

### PR DESCRIPTION
**Pull request by jrothrock**

### CODE REVIEW

**Comment by jrothrock** [comment]
Good point, and yeah that's what I was wondering. I'll push that

File: utils/partition_image.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # This script partitions cropped multi-rock image files into single rock image files.  
-# To run the script, execute "python partition_image.py" 
+# To run the script, execute "python partition_image.py 224"
 
-from scipy.misc import imread, imsave
+# maybe someway to trim whitespace using PIL, https://stackoverflow.com/questions/10615901/trim-whitespace-using-pil
+#     - gradient makes it tough though.
+
```
Comment (kjblakemore) [inline utils/partition_image.py]: I think the whitespace will be fine, the CNN will learn that it is not significant to the classification.

What might be significant are the external contours of the rocks.  How a rock splits might indicate whether it contains copper ore.

**Comment by jrothrock** [comment]
I was more referencing the white area around the “checkerboard” area so we don’t have to crop the image. Then again if we’re going to be taking pictures ourselves, and using the 25 image sets, it may not really matter

**Comment by kjblakemore** [comment]
I understand your idea for the white space, now.

I've added resizing to the master branch.  It always resizes from (772, 463) -> (224, 136).  This maintains the aspect ratio and insures that all the images are the same size when reduced.  Not the most flexible solution, but it will work for preparing our data set and will be easy to change sizes if and when needed.

**Comment by kjblakemore** [comment]
I tried to use your resizing code, but found that the aspect ratio was off a bit.  Plus, we want to make sure that every image is the same size.  That is required for the linear algebra operations in the CNN.

**Comment by jrothrock** [comment]
You're totally right. Casting with int just truncates instead of rounds. I've pulled yours and updated it

**Pull request merged**